chapters/equations.tex

```diff
@@ -858,6 +859,28 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \end{lstlisting}
 \end{example}
 
+\begin{example}
+Resettable continuous time controller initialized either in steady-state or by providing a \lstinline!start! value for state \lstinline!y!:
+\begin{lstlisting}[language=modelica]
+  parameter Boolean steadyState = true;
+  parameter Real y0 = 0 "start and reset value for y, if not steadyState";
+  Real y;
+equation
+  der(y) = a*y + b*u;
+  when { initial(), reset } then
+    if not ( initial() and steadyState ) then
```

Comment:
Personally, I like this airy style, but I'm afraid we should introduce it here and now:
```suggestion
  when {initial(), reset} then
    if not (initial() and steadyState) then
```